app/stac_api/managers.py

```diff
@@ -116,11 +119,16 @@ def _filter_by_datetime_range(self, start_datetime, end_datetime):
     def _parse_datetime_query(self, date_time):
         '''Parse the datetime query as specified in the api-spec.md.
 
-        Returns one of the following
-            datetime, None
-            datetime, '..'
-            '..', datetime
-            datetime, datetime
+        A helper function of filter_by_datetime
+
+        Args:
+            date_time: string
+                Datetime as string (should be in isoformat)
+
+        Returns:
+            queryset filtered by datetime
+
+        Raises: ValidationError
```

Comment:
Short comment to explain why ValidationError can occurs: e.g: 
`
Raises:
    ValidationError: when the date_time string is not a valid isoformat
`